tools/download_tool.py

```
@@ -104,22 +119,21 @@ def main() -> None:
 
     print(f"Downloading {url} to {output}")
     req = urllib.request.Request(url, headers={"User-Agent": "Mozilla/5.0"})
-    with urllib.request.urlopen(req) as response:
-        if url.endswith(".zip"):
-            data = io.BytesIO(response.read())
-            with zipfile.ZipFile(data) as f:
-                f.extractall(output)
-            # Make all files executable
-            for root, _, files in os.walk(output):
-                for name in files:
-                    os.chmod(os.path.join(root, name), 0o755)
-            output.touch(mode=0o755)  # Update dir modtime
-        else:
-            with open(output, "wb") as f:
-                shutil.copyfileobj(response, f)
-            st = os.stat(output)
-            os.chmod(output, st.st_mode | stat.S_IEXEC)
-
+    try:
+        with urllib.request.urlopen(req) as response:
+            download(url, response, output)
+    except urllib.error.URLError as e:
+        if str(e).find("CERTIFICATE_VERIFY_FAILED") == -1:
+            return
```

Let's rethrow the error so the user sees it